.github/workflows/operator-ci.yml

```diff
@@ -50,38 +78,121 @@ jobs:
+          driver: docker
+          start args: '--addons=ingress'
 
       - name: Test operator running in cluster
         working-directory: operator
         run: |
           eval $(minikube -p minikube docker-env)
           mvn clean verify \
-              -Dquarkus.container-image.build=true -Dquarkus.container-image.tag=test \
+              -Dquarkus.container-image.build=true \
               -Dquarkus.kubernetes.deployment-target=kubernetes \
-              -Dquarkus.jib.jvm-arguments="-Djava.util.logging.manager=org.jboss.logmanager.LogManager","-Doperator.keycloak.image=keycloak:${GITHUB_SHA}","-Doperator.keycloak.image-pull-policy=Never" \
+              -Dquarkus.jib.jvm-arguments="-Djava.util.logging.manager=org.jboss.logmanager.LogManager","-Doperator.keycloak.image=${{ secrets.TEST_DOCKER_REGISTRY }}/${{ secrets.TEST_DOCKER_REPOSITORY }}/keycloak:${{ steps.vars.outputs.version }}" \
+              -Dtest.operator.custom.image=${{ secrets.TEST_DOCKER_REGISTRY }}/${{ secrets.TEST_DOCKER_REPOSITORY }}/custom-keycloak:${{ steps.vars.outputs.version }} \
               --no-transfer-progress -Dtest.operator.deployment=remote \
-              -Dtest.operator.kubernetes.ip=$(minikube ip) \
-              -Dtest.operator.custom.image=custom-keycloak:${GITHUB_SHA}
+              -Dtest.operator.kubernetes.ip=$(minikube ip)
+
+  test-olm:
+    name: Test OLM installation
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set outputs
+        id: vars
+        run: echo "::set-output name=version::0.0.1-$(git rev-parse --short HEAD)"
+      - name: Update maven settings
+        run: mkdir -p ~/.m2 ; cp .github/settings.xml ~/.m2/
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: ${{ env.JDK_VERSION }}
+          cache: 'maven'
+      - name: Setup Minikube-Kubernetes
+        uses: manusa/actions-setup-minikube@v2.4.3
+        with:
+          minikube version: v1.24.0
+          kubernetes version: v1.22.3
+          github token: ${{ secrets.GITHUB_TOKEN }}
+          driver: docker
+          start args: '--addons=ingress'
+      - name: Login to docker registry
+        uses: docker/login-action@v1.14.1
+        with:
+          registry: ${{ secrets.TEST_DOCKER_REGISTRY }}
+          username: ${{ secrets.TEST_DOCKER_USERNAME }}
+          password: ${{ secrets.TEST_DOCKER_TOKEN }}
+
+      - name: Build and push the operator image
+        working-directory: operator
+        run: |
+          mvn clean package \
+              -Dquarkus.container-image.build=true \
+              -Dquarkus.container-image.push=true \
+              -Dquarkus.container-image.image="${{ secrets.TEST_DOCKER_REGISTRY }}/${{ secrets.TEST_DOCKER_REPOSITORY }}/keycloak-operator:${{ steps.vars.outputs.version }}" \
+              -Dquarkus.container-image.labels."\"quay.expires-after\""="20h" \
```

Comment:
This label is not injected at the moment due to: https://github.com/quarkusio/quarkus/issues/24820